packages/@o3r/core/src/core/interfaces/configuration.ts

```diff
@@ -36,7 +37,6 @@ export interface CustomConfig<T extends Partial<Configuration> = Partial<Configu
 /**
  * Dynamically Configurable item
  */
-// eslint-disable-next-line no-use-before-define
 export interface DynamicConfigurable<T extends Configuration> {
```

Comment:
Could not support both interface ?
```suggestion
export type DynamicConfigurable<T extends Configuration> = DynamicConfigurableWithSignal<T> | DynamicConfigurableWithObservable<T>;

export interface DynamicConfigurable<T extends Configuration> {
  /**
   * Configuration override
   */
  config: Partial<T> | undefined;
  /**
   * Configuration stream
   */
  config$: Observable<T>;
}
```

Reply:
I tried that in a first time 
but class cannot extends type